tsbridge/config.go

```diff
@@ -124,7 +131,28 @@ func NewConfig(ctx context.Context, opts *ConfigOptions) (*Config, error) {
 		metrics[m.Name] = true
 	}
 
-	log.Debugf(ctx, "Read %d metrics and %d destinations from the config file", len(metrics), len(destinations))
```

Comment:
Any specific reason why you've removed this?

Reply:
It was causing me errors with dev_appserver.py because the context wasn't an app engine context. I'll see if that error continues to be a problem if I add this back.